### ISSUE TITLE
More pythonic way to pass args.

### ISSUE THREAD

**Issue by hmacdope** [choderalab/mtenn#39]
Now that I think about it more pythonic would be to pass these as named args rather than a dict with an expected structure but we can discuss as is big API change.

_Originally posted by @hmacdope in https://github.com/choderalab/mtenn/pull/38#discussion_r1458166955_

**Comment by kaminow** [comment]
yeah this makes sense and shouldn't be too difficult. I had it this way initially because that's how some of  the underlying models expect the input, but I can just as easily package as a dict on the fly as unpackage for the other models

**Comment by hmacdope** [comment]
Fine the way it is TBH
